release-notes/2024-09/README.md

```diff
@@ -0,0 +1,231 @@
+# SystemLink Enterprise release 2024-09 Release Notes
+
+The 2024-09 release for SystemLink Enterprise has been published to <https://downloads.artifacts.ni.com>. This update includes new features, bug fixes, and security updates. Work with your account representative to obtain credentials to access these artifacts. If you are not upgrading from the previous release, refer to past release notes to ensure you have addressed all required configuration changes.
+
+## New Features and Behavior changes
+
+- Navigate to **Operations** » **Schedule** to view the test plans scheduled across all systems in timeline view
+    - [Viewing Scheduled Test Plans](https://www.ni.com/docs/en-US/bundle/systemlink-enterprise/page/viewing-scheduled-test-plans.html).
+
+- Use the following new features when [visualizing data tables in data spaces](https://www.ni.com/docs/en-US/bundle/systemlink-enterprise/page/visualizing-data-tables-in-a-data-space.html).
+    - Search for columns to plot from the axis selection grid.
+    - Color traces by data table and by column metadata.
+
+- Support for [Private Certificate Authorities](https://www.ni.com/docs/en-US/bundle/systemlink-enterprise/page/private-certificate-authorities.html).
+    - Connect to MongoDB and S3 compatible interfaces that use certificates signed by a private authority.
+        - Trusted certificates specified at `global.trustedCertificatesSecrets` are now installed on containers that connect to MongoDB and S3 compatible interfaces. These certificates are used when connecting to these external dependencies.
+        - [View this service configuration](https://github.com/ni/install-systemlink-enterprise/blob/2024-09/getting-started/templates/systemlink-values.yaml#L113)
+    - Call SystemLink APIs from Jupyter Notebook when the SystemLink API endpoint is configured to use a certificate signed by a private authority.
+        - Trusted certificates specified at `global.apiHostCertificateSecret` are now installed on containers that run Jupyter notebooks. These certificates are used when calling SystemLink or other external APIs from notebooks.
```

Comment:
`global.apiHostCertificateSecret` is used only for the certificate deployed from Systems Management to the systems, not for Jupyter notebooks. JupyterHub & Notebook Executions use the same `global.trustedCertificatesSecrets` that MongoDB and S3 use

Reply:
Good catch. Updated.